packages/@react-spectrum/tag/stories/TagGroup.stories.tsx

```diff
@@ -91,7 +91,19 @@ storiesOf('TagGroup', module)
         }
       </TagGroup>
     )
-  );
+  )
+  .add('maxRows: 2', () => (
```

Comment:
I want controls to test this with truncation (I know, narrow enough single column they truncate) and onRemove. Waiting for CSF 3 refactor. :)

Reply:
Yeah, I think it might be worth going ahead and updating this to CSF3, and also adding some chromatic stories in this PR